fastlane/metadata/it/description.txt

```diff
@@ -0,0 +1,11 @@
+Con un download puoi aiutare a combattere il cambiamento climatico, tutelando anche la tua privacy. Scarica subito la app Ecosia per piantare alberi effettuando ricerche, è completamente gratuita!
```

Comment:
I see that we support Italian, though our links / blog don't anymore. Bit off context question but worth highlighting for consistency between apps and browser experience